app/cdap/components/ParsingDrawer/styles.ts

```diff
@@ -0,0 +1,192 @@
+    importSchemaTextStyles: {
+      fontFamily: 'Noto Sans',
+      fontStyle: 'normal',
+      fontWeight: 400,
+      fontSize: 14,
+      lineHeight: '150%',
+      letterSpacing: '0.15px',
+      color: '#000000',
+    },
+    dividerLineStyles: {
+      width: 1,
+      height: 28,
+      backgroundColor: '#DADCE0',
+      margin: '0 15px',
+    },
+    checkboxStyles: {
+      display: 'flex',
+      width: '100%',
+      marginBottom: 0,
+    },
+    labelTextStyles: {
+      fontFamily: 'Noto Sans',
+      fontStyle: 'normal',
+      fontWeight: 400,
+      fontSize: 14,
+      lineHeight: '150%',
+      letterSpacing: '0.15px',
+      color: '#5F6368',
+    },
+    formFieldWrapperStyles: {
+      width: 'calc(100% - 60px)',
+      marginRight: '60px',
+    },
+    marginBottomStyles: {
+      marginBottom: '10px',
+    },
+    selectFieldStyles: {
+      height: '40px',
+      background: '#FFFFFF',
+      border: '1px solid #DADCE0',
+      borderRadius: '4px',
+      padding: '5px 15px 0px 15px',
+      fontFamily: 'Noto Sans',
+      fontSize: '14px',
+      '&:before': {
+        display: 'none',
+      },
+      '&:focus-visible': {
+        outline: 'none !important',
+      },
+      '&:after': {
+        display: 'none',
+      },
+      '& .MuiInputBase-input': {
+        padding: '6px 0px 11px',
+      },
+    },
+    optionStyles: {
+      fontFamily: 'Noto Sans',
+      fontSize: '14px',
+      lineHeight: '150%',
+      letterSpacing: '0.15px',
+      color: '#000000',
+    },
+    selectIconStyles: {
+      top: 'calc(50% - 10px)',
+      right: '10px',
+    },
+    selectStyles: {
+      '&:focus': {
+        'background-color': 'transparent',
+      },
+    },
+    underlineStyles: {
+      display: 'none',
+    },
+    bodyContainerStyles: {
+      display: 'flex',
+      justifyContent: 'space-between',
+      flexDirection: 'column',
+      height: 'calc(100% - 120px)',
+    },
+    infoWrapperStyles: {
+      display: 'flex',
+      alignItems: 'center',
+    },
+    bottomSectionStyles: {
+      display: 'flex',
+      flexDirection: 'column',
+    },
+    infoTextStyles: {
+      fontFamily: 'Noto Sans',
+      fontStyle: 'normal',
+      fontWeight: 400,
+      fontSize: '14px',
+      lineHeight: '150%',
+      letterSpacing: '0.15px',
+      color: '#5F6368',
+      opacity: '0.8',
+      marginLeft: '10px',
+    },
+    applyButtonStyles: {
+      width: '162px',
+      height: '36px',
+      background: '#3994FF',
+      boxShadow: '0px 2px 4px rgba(70, 129, 244, 0.15)',
+      borderRadius: '4px',
+      fontFamily: 'Noto Sans',
+      fontStyle: 'normal',
+      fontWeight: 400,
+      fontSize: '15px',
+      lineHeight: '26px',
+      letterSpacing: '0.46px',
+      color: '#FFFFFF',
+      alignSelf: 'flex-end',
+      marginTop: '30px',
+      textTransform: 'none',
+    },
+    MUIPopover: {
+      left: '472px !important',
+      minWidth: '345px !important',
+      top: '253px !important',
+    },
```

Comment:
use number directly instead of using number with px everywhere
and use MUI colors if possible